images/build/Dockerfiles/gitHubRunners.BuildPackDepsStretch.Dockerfile

```diff
@@ -1,4 +1,4 @@
 # The digest here must match with what is present in the following location
 # so that downloading of the image is very fast in GitHub workflows
 # https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md
-FROM buildpack-deps:stretch@sha256:23c6d55a4d5ad7c9476638196920d67a97702433b5b48465496af6ab3214e7e4
+FROM buildpack-deps:stretch@sha256:d95db8b9293c71d0f9b6a12d96d1ace65af81fd6535e5cb07078df85b3147a76
```

Comment:
again?

Reply:
should we just create another githubrunner image that always points to latest  buildpackdepsstretch so that it will ensure all security patches ... we can keep this one for github actions specifically

Reply:
> again?

If they are building their VMs regularly then yeah this will keep happening.

> should we just create another githubrunner image that always points to latest buildpackdepsstretch so that it will ensure all security patches ... we can keep this one for github actions specifically

Yeah, let me think about it.